client/my-sites/plans-features-main/index.jsx

```diff
@@ -325,6 +327,8 @@ class PlansFeaturesMain extends Component {
 
 		return (
 			<div className="plans-features-main">
+				<QueryPlans />
+				<QuerySitePlans siteId={ site.ID } />
```

Comment:
site isn't required here, so it'd be safer to pass through the id directly or at least use lodash `get` to avoid an error